components/v1/PortofolioPage/PortofolioPage.tsx

```diff
@@ -159,13 +171,23 @@ const PortofolioPage: FunctionComponent<PortofolioPageProps> = ({}) => {
                                                 <tr className="text-right text-sm font-semibold">
                                                     <td className="flex items-center space-x-4 text-left">
                                                         <img className="h-[40px] w-[40px]" src={metadata.vaultLogo} alt={metadata.title} />
-                                                        <p className="text-gray-light-12 dark:text-gray-dark-12">{metadata.vaultTitle}</p>
+                                                        <div className="flex flex-col space-y-[8px]">
+                                                            <p className="text-gray-light-12 dark:text-gray-dark-12">{metadata.vaultTitle}</p>
+                                                            <p className="text-gray-light-10 dark:text-gray-dark-10 lg:hidden">
+                                                                {formattedRvETHUSDCBalance.toFixed(2)} {metadata.vaultTitle}
+                                                            </p>
+                                                        </div>
+                                                    </td>
+                                                    <td className="hidden text-gray-light-10 dark:text-gray-dark-10 lg:table-cell">
+                                                        {formattedRvETHUSDCBalance.toFixed(2)} {metadata.vaultTitle}
                                                     </td>
-                                                    <td className="text-gray-light-10 dark:text-gray-dark-10">
-                                                        {formattedRvEthriseUsdcBalance.toFixed(2)} {metadata.vaultTitle}
+                                                    <td className="hidden text-green-light-11 dark:text-green-dark-11 lg:table-cell">+$13.34</td>
+                                                    <td>
+                                                        <div className="flex flex-col space-y-[8px]">
+                                                            <p className="text-gray-light-12 dark:text-gray-dark-12">{dollarFormatter.format(latestVaultExchangeRateFormatted * formattedRvETHUSDCBalance)}</p>
```

Comment:
@math-marcellino Please use raw value instead of fomatted one (`exchangeRate * rvETHUSDCBalance`)